packages/start/src/runtime/server-runtime.ts

```diff
@@ -55,7 +58,7 @@ class SerovalChunkReader {
     // deserialize the data
     const bytes = Number.parseInt(this.buffer.substring(1, 11), 16); // ;0x00000000;
```

Comment:
@hardywu does the substring still work for special characters? Would like to know.

Reply:
substring fn works by charater. Special characters are treated the same as others.  So yes. It still works.